### DISCUSSION TITLE
Unable to create airflow user

### DISCUSSION THREAD

**Discussion by sinjup** [apache/airflow#25973] [Q&A]
Currently I am using airflow version 2.2.5 and wish to upgrade to 2.3.x , while using versions 2.3+ i am unable to create a user. It's showing "No user created use fab command to create" I have added user creation commands in my entrypoint.sh file.
![image](https://user-images.githubusercontent.com/110447163/186823735-b972c4c9-cced-4902-8d07-d17d0d3c37ee.png)

**Comment by potiuk** [comment]
You should not need to create a user if you upgrade. It should be there already.
